### ISSUE TITLE
Unify source application icons

### ISSUE THREAD

**Issue by ztefanie** [digitalfabrik/entitlementcard#1825]
**Is your feature request related to a problem? Please describe.**
Currently we are displaying the verein360 and juleica

![image](https://github.com/user-attachments/assets/a928f509-11ff-4762-85f1-494e873ee453)

**Describe the solution you'd like**
Unify the icons so the ui looks more clean

**Describe alternatives you've considered**
Keep everything as it is

**Comment by hauf-toni** [comment]
> My suggestion would be to use a chip component or sth similar but its up to the design team https://mui.com/material-ui/react-chip/

I like that. Just to check—you meant that Juleica wants to keep the logo in there, right @connium ?

**Comment by connium** [comment]
@hauf-toni No, I don't think it is necessary to use any logos at all. Chips would be totally fine from my point of view.

**Comment by hauf-toni** [comment]
Design can be found [📌 here](https://www.figma.com/design/SDoGnXIjCCXpE1tAAJkRVc/entitlementcard?node-id=1382-4535&t=Oex4X81aLKdNDRkC-1) @f1sh1918 @andrew8er

**Comment by f1sh1918** [comment]
For me its good to go
